CONTRIBUTING.md

```diff
@@ -62,6 +62,9 @@ idpbuilder get secrets
 ```
 
 As described in the main readme file, the above command is equivalent to running:
+```bash
+kubectl get secrets -n gitea gitea-credential -o go-template='{{ range $key, $value := .data }}{{ printf "%s: %s\n" $key ($value | base64decode) }}{{ end }}'
+```
```

Comment:
This is my bad. I missed copy pasta here.

```suggestion
```bash
kubectl -n argocd get secret argocd-initial-admin-secret
kubectl get secrets -n gitea gitea-admin-secret
kubectl get secrets -A -l cnoe.io/cli-secret=true
```
```